### ISSUE TITLE
Save-LatestUpdate does not work for WindowsVersion = Windows7

### ISSUE THREAD

**Issue by DominikBritz** [aaronparker/LatestUpdate#27]
# Expected Behavior

"Get-LatestUpdate -WindowsVersion Windows7 | Save-LatestUpdate" should download updates

# Current Behavior

```
Join-Path : Cannot convert 'System.Object[]' to the type 'System.String' required by parameter 'ChildPath'. Specified
method is not supported.
At C:\Program Files\WindowsPowerShell\Modules\LatestUpdate\2.3.1.81\Public\Save-LatestUpdate.ps1:83 char:39
+             $target = Join-Path $Path $filename
+                                       ~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Join-Path], ParameterBindingException
    + FullyQualifiedErrorId : CannotConvertArgument,Microsoft.PowerShell.Commands.JoinPathCommand
```

This fails only for Windows 7. Windows 8 and 10 works fine.

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* **LatestUpdate Version**: 2.3.1.81
* **PowerShell Version**: 5.1.14393.2758
* **Operating System**: Server 2016

**Comment by DominikBritz** [comment]
I think this is related to [issue 23](https://github.com/aaronparker/LatestUpdate/issues/23). As the URL property contains an string for an exe and a msu, it is not a string anymore but an array. Join-Path expects a string hence the error

**Comment by aaronparker** [comment]
What's the preferred update for Windows 7 - MSU like Windows 10 and 8 or EXE?

**Comment by DominikBritz** [comment]
MSU!

**Comment by aaronparker** [comment]
Fixed in commit 39b003c637152214b59a71158ffbac9774d8d395

**Comment by aaronparker** [comment]
Fixed in release v2.4.0.89.
